docs/release-notes/deploy-local-auto-password.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**Security Fixes**
+
+-  CLI: If no user accounts have already been created, and neither
+   ``security.initial_user_password`` in master.yaml nor the ``--initial-user-password`` CLI flag is
+   present when running ``det deploy local`` with the ``master-up`` or ``cluster-up`` commands, an
+   initial password will be generated and displayed to the user.
```

Comment:
@jesse-amano-hpe does the system display the option to change the auto generated password? Also, we should probably clarify this is "when deploying locally" (my fault)....

e.g.,

:orphan:

**Security Fixes**

-  CLI: When deploying locally using ``det deploy local`` with ``master-up`` or ``cluster-up`` commands and no user accounts have been created yet, an initial password will be automatically generated and shown to the user (with the option to change it) if neither  ``security.initial_user_password`` in ``master.yaml`` nor the ``--initial-user-password`` CLI flag is
   present.